tox.ini

```diff
@@ -10,7 +10,7 @@ commands = pytest test --junit-xml pkr-test-report-{envname}.xml --junit-prefix
 
 [testenv:pylint]
 deps = pylint
-commands = bash -c \'{envbindir}/pylint -f parseable -r no --py3k {toxinidir}/pkr\'
+commands = pylint -f parseable -r no --py3k {toxinidir}/pkr
```

Comment:
removing this breaks on some envs

Reply:
Actually, it is broken for me with this. Not sure why, launching tox from inidir.

Get : 
```
pkr\:1: [F0001(fatal), ] No module named pkr\
```